resources/zib/ext-DispenseRequest.DispenseLocation.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="ext-DispenseRequest.DispenseLocation" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/ext-DispenseRequest.DispenseLocation" />
+  <name value="ExtDispenseRequestDispenseLocation" />
+  <title value="ext DispenseRequest DispenseLocation" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="email" />
+      <value value="info@nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="An extension to provide a reference to the Location resource to indicate where the medication is (to be) dispensed." />
+  <purpose value="This extension represents DispenseLocation of [DispenseRequest](https://zibs.nl/wiki/DispenseRequest-v1.0.3(2020EN))" />

```

Comment:
I'd say: "the DispensoLocation of zib DispenseRequest v1.0.3 (2020)"

Reply:
Adjusted, thnx!